tools/lambda-promtail/lambda-promtail/cw.go

```diff
@@ -38,7 +38,7 @@ func parseCWEvent(ctx context.Context, b *batch, ev *events.CloudwatchLogsEvent)
 }
 
 func processCWEvent(ctx context.Context, ev *events.CloudwatchLogsEvent) error {
-	batch := newBatch()
+	batch, _ := newBatch(ctx)
```

Comment:
We should not swallow this error. Perhaps `newBatch` shouldn't accept entries and then `newBatch` won't need to return an error.

Reply:
since the batch will always be empty here it should never fail, thats why I ignored it